leaflet-draw/build.boot

```diff
@@ -20,17 +20,17 @@
 
 (deftask package []
   (comp
-    (download :url      (format "https://github.com/Leaflet/Leaflet.draw/archive/%s.zip" +lib-version+)
-              :checksum "BE611BFB85008C3C964ABE46C3C751D2"
-              :unzip    true)
-    (sift :move {#"^Leaflet.draw-(.*)/dist/leaflet.draw-src.js"    	"cljsjs/development/leaflet-draw.inc.js"
-                 #"^Leaflet.draw-(.*)/dist/leaflet.draw.js"        	"cljsjs/production/leaflet-draw.min.inc.js"
-                 #"^Leaflet.draw-(.*)/dist/leaflet.draw.css"       	"cljsjs/common/leaflet-draw.inc.css"
-                 #"^Leaflet.draw-(.*)/dist/images/spritesheet.png" 	"cljsjs/common/images/spritesheet.png"
-                 #"^Leaflet.draw-(.*)/dist/images/spritesheet-2x.png" "cljsjs/common/images/spritesheet-2x.png"
-                 }) ;; TODO Provide one-liner for images
+    (download :url      (format "https://registry.npmjs.org/leaflet-draw/-/leaflet-draw-%s.tgz" +lib-version+)
+              :decompress true
+              :compression-format "gz"
+              :archive-format "tar")
+    (sift :move {#"^.*/dist/leaflet.draw-src.js" "cljsjs/development/leaflet-draw.inc.js"
```

Comment:
Would be good time to update this package to use convention of adding library name to the path: https://github.com/cljsjs/packages/wiki/Creating-Packages#3-create-a-directory-structure-for-the-package

Especially as this package now uses paths like `cljsjs/common/images/spritesheet.png`, which could easily clash with other package (which doesn't include lib name on path).

Reply:
Ok, I'll change paths to something like `cljsjs/leaflet_draw/common/images/*`.
Should we use the dash->underscore convention?

Reply:
I think it is not necessary to change dashes to underscore. I think other packages just use dashes: https://github.com/cljsjs/packages/blob/master/leaflet-fullscreen/build.boot#L25